src/og/utils/shared.js

```diff
@@ -940,4 +940,21 @@ export function cloneArray(items) {
  */
 export function isUndef(obj) {
     return obj === void 0;
+}
+
+/**
+ * Promise for load images
+ * @function
+ * @param {string} url - link to image.
+ * @returns {Promise<Image>} Returns promise.
+ */
+export async function loadImage(url) {
```

Comment:
Btw, we have loader(Loader)for images and binary data etc, so might be better change it in future.